pallets/funding/src/lib.rs

```diff
@@ -1139,21 +1144,37 @@ pub mod pallet {
 
 		#[pallet::call_index(26)]
 		#[pallet::weight(Weight::from_parts(1000, 0))]
-		pub fn migrate_one_participant(
+		pub fn send_pallet_migration_for(
 			origin: OriginFor<T>,
 			project_id: ProjectId,
 			participant: AccountIdOf<T>,
 		) -> DispatchResult {
 			let _caller = ensure_signed(origin)?;
-			Self::do_migrate_one_participant(project_id, participant)
+			Self::do_send_pallet_migration_for(project_id, participant)
 		}
 
 		#[pallet::call_index(27)]
 		#[pallet::weight(Weight::from_parts(1000, 0))]
-		pub fn confirm_migrations(origin: OriginFor<T>, query_id: QueryId, response: Response) -> DispatchResult {
+		pub fn confirm_pallet_migrations(
+			origin: OriginFor<T>,
+			query_id: QueryId,
+			response: Response,
+		) -> DispatchResult {
 			let location = ensure_response(<T as Config>::RuntimeOrigin::from(origin))?;
 
-			Self::do_confirm_migrations(location, query_id, response)
+			Self::do_confirm_pallet_migrations(location, query_id, response)
+		}
+
+		#[pallet::call_index(38)]
+		#[pallet::weight(Weight::from_parts(1000, 0))]
```

Comment:
Can we add a TODO for these dummy weights so we remember to change them when these calls are enabled?

Reply:
Will do on the child PR that reorgs the extrinsics